From c749b2698d3a9938c1116311910b9acca8db0217 Mon Sep 17 00:00:00 2001
From: Alexandre Schnegg <alexandre.schnegg@sonova.com>
Date: Mon, 16 Feb 2026 13:46:58 +0100
Subject: [PATCH] Change the greeting message

---
 src/hello.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/hello.c b/src/hello.c
index 453962f..7e8c3dd 100644
--- a/src/hello.c
+++ b/src/hello.c
@@ -57,7 +57,7 @@ main (int argc, char *argv[])
 #endif
 
   /* Having initialized gettext, get the default message. */
-  greeting_msg = _("Hello, world!");
+  greeting_msg = _("Hello, world from patched code !");
 
   /* Even exiting has subtleties.  On exit, if any writes failed, change
      the exit status.  The /dev/full device on GNU/Linux can be used for
